src/home/useGroupCallRooms.ts

```diff
@@ -87,23 +84,22 @@ export function useGroupCallRooms(client: MatrixClient): GroupCallRoom[] {
         return;
       }
 
+      // XXX: This should use client.matrixRTC.getActiveRoomSessions() not the old
+      // group call stuff!
```

Comment:
Mmm, not quite Dave, because that would only return sessions with people participating in them, which is not what we want, is it? What exactly *do* we want here? Just all the rooms, probably.